specs/regional_communities.md

```diff
@@ -1,61 +1,333 @@
 - Requirements, assumptions and dependencies (internal and external).
 - Outline identified potential problems and roadblocks related to building the solution.
 - Optionally: Budget limit.
 
-### Backwards Compatibility
+### Scope
+<!-- Define the scope and potential phases of the widget -->
+In scope
+- All geographies
+- All languages
+- All verticals
+- All human users interested in joining, forming or managing communities of likeminded
+
+Typical resons to come together in a community:
+- Networking
+- Learning from each other, discussing
+- Support and help each other
+- Find new friends, build meaningful relationship
+
+What is NOT within the scope of regional communities? Groups that are not bulding lasting relationships, groups where the only purpose is to facilitate one-time or temporary transactions.
+- Buy and sell groups
+- Job listing / job search
+- Gig economy
 
+### Expected Functionality
+<!-- What are the Minimal Viable Requirements (MV)  the widget should meet to be considered complete? -->
+The minimum requirements the widget should meet include:
+- Built on BOS / Alpha / Near.social
+- Can have a single admin owner, several owners / admins, or a DAO as an owner
+- Members can post messages within community
+- Messages posted within communities that I follow will show up in my stream
+- Supports all three dimensions
+   - Geography as a hierarchy (each community has one geography, and has a parent with a higher level geography)
+   - Language as a flat list, single-select
+   - Consteallation as tags, multi-select
+- Allows the owner to split the community on one of the dimensions
+- Allows the owner to reassign ownership fully, and allows the owner(s) to give ownership rights to others
+
+### Backwards Compatibility
 All proposals that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The doc must explain how to deal with these incompatibilities or the set of requirements to for the backward compatibility to be discussed in a submission. Submissions without a sufficient backwards compatibility treatise may be rejected outright.
 
+## Phases
+<!-- Do the project have multiple phases? Identify a high-level summary of each phase. -->
+
+### Phase 1: MVP
+- The app owner can create global parent communities for each Constellation
+- Users can find their community, assign them selves membership permissionlessly (similar to "follow")
+- Members can start posting messages to the Community board.
+- Members can perform a sibling split along one of the three dimensions
+- Members can perform a child split along one of the three dimensions, 
+- Member executing the split will become the owner of the new Community
+
+### Phase 2: Admin Functions
+- Admins can assign other admins
+- Users can request core memberships
+- Admins can approve core memberships
+- Admins can warn, silence and ban users
```

Comment:
With silence they can still see what's going on. With ban they cannot (if it's a closed membership community). For open communities it might be the same. 

Yes, we should consider an appeal process and an un-ban feature.